Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM calico/build:latest
+MAINTAINER projectcalico
+
+ADD . /code
+WORKDIR /code
+
+ARG K8S_VERSION=local
```

Comment:
Could you explain what the K8S_ARG might be used for? In general, we expect CNI releases to apply across Kubernetes versions.